open_humans/views.py

```diff
@@ -516,6 +517,63 @@ def form_valid(self, form):
         return super(ActivityMessageFormView, self).form_valid(form)
 
 
+class StatisticView(NeverCacheMixin, SourcesContextMixin, TemplateView):
+    """
+    Show latest statistics on signed up users/projects etc.
+    """
+
+    template_name = 'pages/statistics.html'
+
+    @staticmethod
+    def get_number_member():
+        members = Member.objects.all()
+        members_with_data = members.annotate(
```

Comment:
I think this is equivalent if you just need any member with > 0 data files: 

```python
members_with_data = members.filter(user__datafiles__isnull=False).count()
```

Reply:
I followed @madprime's lead here from https://github.com/OpenHumans/open-humans/blob/master/open_humans/management/commands/stats_new.py 🙂

Reply:
I probably didn't know about (or remember) `isnull` :)